testcases/OpalUtils.py

```diff
@@ -186,7 +186,7 @@ def flash_program_guard_part(self):
     def runTest(self):
         self.l_dic = []
         self.utils_init()
-        self.c = self.cv_SYSTEM.sys_get_ipmi_console()
+        self.c = self.cv_SYSTEM.cv_HOST.get_ssh_connection()
```

Comment:
This as well, we need to use console instead of SSH.

Reply:
Fixed